docs/source/getting-started/setting-up-hypervisor-driver-plugin.adoc

```diff
@@ -1,35 +1,34 @@
-[[docker-machine-driver-install]]
-= Installing Docker Machine drivers
+[[set-up-hypervisor-driver-plugin]]
+= Setting up the hypervisor driver plugin
 :icons:
 :toc: macro
 :toc-title:
 :toclevels: 1
 
 toc::[]
 
-[[docker-machine-drivers-overview]]
+[[hypervisor-drivers-overview]]
 == Overview
-
 Minishift uses Docker Machine and its driver plugin architecture to
-provide a consistent way to manage the OpenShift VM. Minishift embeds
-VirtualBox and VMware Fusion drivers so no additional steps are required to use them.
+provide a consistent way to manage the OpenShift VM.
+Minishift embeds VirtualBox and VMware Fusion drivers so no additional steps are required to use them.
 
-However, other drivers require manual installation steps that are
+However, other drivers require manual installation and setup that are
 described in the following sections.
 
 [[kvm-driver-install]]
 == KVM driver
 
 Minishift is currently tested against `docker-machine-driver-kvm` version 0.7.0.
 
-Install and execute the KVM binary as follows:
-
+. Install the KVM binary as follows:
++
 ----
 $ sudo curl -L https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.7.0/docker-machine-driver-kvm -o /usr/local/bin/docker-machine-driver-kvm
 $ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
 ----
 
-For more information, see the GitHub documentation of the https://github.com/dhiltgen/docker-machine-kvm#quick-start-instructions[docker machine KVM driver].
+. Execute the KVM binary, for more information, see the GitHub documentation of the https://github.com/dhiltgen/docker-machine-kvm#quick-start-instructions[docker machine KVM driver].
```

Comment:
@Preeticp I think, putting `Execute the KVM binary` here is wrong here. 

I was thinking of this: 
. Install the KVM binary as follows:
```
$ sudo curl -L https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.7.0/docker-machine-driver-kvm -o /usr/local/bin/docker-machine-driver-kvm
```
. Make the binary executable as follows:
```
$ sudo chmod +x /usr/local/bin/docker-machine-driver-kvm
```
For more information, see the GitHub documentation of the [docker machine KVM driver](https://github.com/dhiltgen/docker-machine-kvm#quick-start-instructions).